README.md

```diff
@@ -34,3 +34,4 @@ Make company global travel and accommodation easy and convenient for the strong
     npm run start
   ```
 
+- You need to get a .coveralls.yml file locally for code coverage to get started
```

Comment:
@davidfrank96 I see that you've added a message here. This should have a title and also explain how to go about getting the token to put in the `.coveralls.yml` file

Reply:
Done..